html/semantics/document-metadata/the-link-element/stylesheet-not-removed-until-next-stylesheet-loads.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<link rel="author" title="Dom Farolino" href="mailto:dom@chromium.org">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/links.html#link-type-stylesheet">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var t = async_test("Check that the style sheet loaded by an <link> element " +
+                   "is available to scripts until its successor is completely loaded");
+</script>
+<link href="style.css?pipe=trickle(d3)" rel="stylesheet" id="style_test"
+      onerror="t.step(function() { assert_unreached('Sheet should load OK'); });"></link>
+<script>
+  t.step(function() {
+    assert_true(document.styleSheets.length === 1 &&
+                document.styleSheets[0].href.includes("style.css"),
+                "The style sheet 'style.css' is not available to scripts");
```

Comment:
Generally, the assertion messages should be stated as things that should happen (like in your other tests), not as things that should not happen. This is inconsistent across WPT but especially for code readers I think it's the right thing to do.